cc1120_arduino_tests/cc1120_spi.h

```diff
@@ -0,0 +1,203 @@
+#ifndef SPI_CC1120_H
+#define SPI_CC1120_H
+
+#include <stdint.h>
+#include <stdbool.h>
+
+#define R_BIT 1 << 7
+#define BURST_BIT 1 << 6
+
+struct cc_status {
+  uint8_t res : 4;
+  uint8_t state : 3;
+  uint8_t chip_ready : 1;
+};
+
+union cc_st {
+  struct cc_status ccst;
+  uint8_t data;
+};
+
```

Comment:
Define the part number (0x48) and part version number. It should be helpful for testing reads from the extended register space. 
Something like:
#define CC1120_PART_NUM  0x48U;
#define CC1120_PART_VER  ?; // Idk what the part version is lol

Reply:
Or you can make a struct to hold this info

Reply:
This is all defined in the cc1120_regs file, is that fine?

Reply:
I don't see them there. The extended addresses (CC1120_REGS_EXT_PARTNUMBER and CC1120_REGS_EXT_PARTVERSION) are stored in that file. What I'm saying is we should also define the values for the data actually present at those addresses. For example, when we read from the register with the following 8-bit address: CC1120_REGS_EXT_PARTNUMBER, the data we get back should be 0x48 (An 8-bit number that depends on which transceiver it is. Since we're using the CC1120, the user's guide says the value is 0x48). 

Also, since the values I'm talking about aren't addresses, they shouldn't be stored in the regs file. It'd make more sense to store them in the cc1120_spi.h file.